composer.json

```
@@ -38,7 +39,7 @@
     },
     "scripts": {
         "test": "vendor/bin/phpunit",
-        "pint": "vendor/bin/pint"
+        "pint": "vendor/bin/pint --config ./pint.json"
```

`./` has no meaning here.

```suggestion
        "pint": "vendor/bin/pint --config pint.json"
```